src/index.ts

```diff
@@ -39,23 +42,25 @@ app.engine(
   }),
 )
 
-registerHealthRoute(app)
-registerLoginRoute(app)
-registerRecoveryRoute(app)
-registerRegistrationRoute(app)
-registerSettingsRoute(app)
-registerVerificationRoute(app)
-registerSessionsRoute(app)
-registerWelcomeRoute(app)
-registerErrorRoute(app)
+registerHealthRoute(router)
+registerLoginRoute(router)
+registerRecoveryRoute(router)
+registerRegistrationRoute(router)
+registerSettingsRoute(router)
+registerVerificationRoute(router)
+registerSessionsRoute(router)
+registerWelcomeRoute(router)
+registerErrorRoute(router)
 
-app.get("/", (req: Request, res: Response) => {
+router.get("/", (req: Request, res: Response) => {
   res.redirect(303, "welcome")
 })
 
-registerStaticRoutes(app)
-register404Route(app)
-register500Route(app)
+registerStaticRoutes(router)
+register404Route(router)
+register500Route(router)
+
+app.use(baseUrl, router)
```

Comment:
okay makes sense, I would need to check this out more thoroughly since this application is re-used to create the Ory Network Account Experience.